LOG/src/main/java/xplor/LOG.kt

```diff
@@ -184,7 +182,7 @@ object LOG {
         if (LOGconfig.isEnabled) {
             val message = message()
             log(level = ERROR) { makeLogItem(t, message) }
-            sentryCallback(t, ERROR, message)
+            theSentryCallback(t, ERROR, message)
```

Comment:
no, the only reason is that 'sentryCallback' starts with the letter 's'  --- our removeLOG.sh that runs during a release build comments out all LOG.s code .... so that was commenting out the LOG.sentryCallback = ...
I changed it to start with a different letter so that our remove code would not comment out the sentryCallback.